NorthstarDLL/mods/modmanager.cpp

```diff
@@ -56,6 +56,14 @@ Mod::Mod(fs::path modDir, char* jsonBuf)
 
 	Name = modJson["Name"].GetString();
 
+	// Don't load blacklisted mods
+	// TODO [Fifty]: Add a launch arg to skip this check
```

Comment:
Should we maybe set this to draft until implemented?

Reply:
This is needed now and is trivial to change to a different system later

Reply:
Done